app/assets/javascripts/kassi.js

```diff
@@ -722,17 +722,28 @@ function initialize_accept_transaction_form(
   }
 }
 
-function updateSellerGetsValue(priceInputSelector, youWillGetSelector, currencySelector, communityCommissionPercentage, minCommission) {
-  $display = $(youWillGetSelector);
+function updateSellerGetsValue(priceInputSelector, displayTargetSelector, currencySelector, communityCommissionPercentage, minCommission, showReversed) {
```

Comment:
Now when we are doing a bigger refactoring and new functionality related to this, should we move this to own js module? We had the goal of getting rid of one big kassi.js file...